tests/unit/execution_tree/primitives/add_operation.cpp

```diff
@@ -3,14 +3,16 @@
 //   Distributed under the Boost Software License, Version 1.0. (See accompanying
 //   file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+#ifndef NOMINMAX
```

Comment:
This can be moved to `<phylanx/config.hpp>` for now. Also, this should be defined only if `_MSC_VER` is defined